frontend/src/metabase/visualizations/lib/chart_values.js

```diff
@@ -320,7 +316,7 @@ export function onRenderValueLabels(
     addLabels(sample, compactForSeries[index]);
     const totalWidth = chart
       .svg()
-      .selectAll(".value-label-outline")
+      .selectAll(".value-label, .value-label-white")
```

Comment:
Since the `text` with the `value-label-outline` class was ever present, we used it to calculate this `totalWidth` value. Now, we just need to select whichever of `value-label` or `.value-label-white` is present.